src/components/ContentSidebar/ContentSidebar.js

```diff
@@ -522,14 +529,48 @@ class ContentSidebar extends PureComponent<Props, State> {
     };
 
     /**
-     * File tasks fetch success callback
+     * Update Tasks to include task assignments
      *
      * @private
-     * @param {Object} tasks - Box task
-     * @return {void}
+     * @param {Array<TaskAssignment>} entries - Box task assignment entries
+     * @param {TaskAssignments} assignments - Box task assigments
+     * @return {TaskAssignments} Updated Box task assignments
+     */
+    populateTaskAssignments(entries: Array<TaskAssignment>, assignments: TaskAssignments): TaskAssignments {
```

Comment:
Can you make the params a little more descriptive? Somehow indicate that entries are coming from the original tasks call and these assignments are what we are going to be pushing in.

Reply:
Was there a reason you didn't want to pass the whole task object instead of just the entries array?

Reply:
This is updating the assignments within a specific task. Otherwise would have to loop through again to figure out which task is being updated